dotCMS/src/main/java/com/dotcms/rest/api/v1/folder/FolderResource.java

```diff
@@ -106,4 +116,216 @@ public final Response loadFolderByURI(@Context final HttpServletRequest httpServ
         return response;
     }
 
+    @GET
+    @Path ("/sitename/{siteName}/folder/{folder : .+}")
+    @JSONP
+    @NoCache
+    @Produces({MediaType.APPLICATION_JSON, "application/javascript"})
+    public final Response loadFolderChildrenByURIPath(@Context final HttpServletRequest httpServletRequest,
+                                                      @Context final HttpServletResponse httpServletResponse,
+                                                      @PathParam("siteName") final String siteName,
+                                                      @PathParam("folder") final String uri){
+        Response response = null;
+        final InitDataObject initData = this.webResource.init(null, httpServletRequest, httpServletResponse, true, null);
```

Comment:
you can use the InitBuilder

Reply:
I am using the same code that is in the method loadFolderByURI above this one.